2019-02-01-preview

jimmyca15 opened the pull request:
This is for the Microsoft.AppConfiguration RP. Our swagger was already reviewed and approved here: https://github.com/Azure/azure-rest-api-specs-pr/pull/725

jimmyca15 committed:
2019-02-01-preview

AutorestCI commented:
# Automation for azure-sdk-for-go

AutorestCI commented:
# Automation for azure-sdk-for-js

AutorestCI commented:
# Automation for azure-sdk-for-ruby

AutorestCI commented:
# Automation for azure-sdk-for-python

nschonni commented on specification/appconfiguration/resource-manager/Microsoft.AppConfiguration/preview/2019-02-01-preview/appconfiguration.json:
```diff
@@ -0,0 +1,866 @@
+      }
+    },
+    "ConfigurationStore": {
+      "description": "The configuration store along with all resource properties. The Configuration Store will have all information to begin utlizing it.",
```
```suggestion
      "description": "The configuration store along with all resource properties. The Configuration Store will have all information to begin utilizing it.",
```

azuresdkci commented:
Can one of the admins verify this patch?

AutorestCI commented:
# Automation for azure-sdk-for-java

veronicagg reviewed (approved):
Thanks for letting me know this is from Private repo pr, already reviewed and approved.

veronicagg commented on specification/appconfiguration/resource-manager/readme.md:
```diff
@@ -0,0 +1,89 @@
+  - repo: azure-sdk-for-java
+  - repo: azure-sdk-for-net
+  - repo: azure-sdk-for-js
+  - repo: azure-sdk-for-node
```
could you update this like to be azure-sdk-for-ruby instead of node?
I see that ruby is missing, and the lines below are after_scripts for ruby. Node is getting deprecated, so no need to include it.

nschonni commented:
Some of the build failures are from the BOM at the start of the files

jimmyca15 committed:
Added ruby spec. Removed Node spec. Fixed spelling error in 'utilizing'. Removed BOM from the beginning of example files.

jimmyca15 commented:
@veronicagg I removed the node spec and added the ruby spec as suggested. 

@nschonni thanks for finding that spelling error, I corrected it. I also removed the BOM from the example files since you suggested it might be causing the build to fail.

veronicagg reviewed (approved):
Thanks for addressing the comments! 
Are the APIs deployed already? If so, I can merge the PR.

jimmyca15 commented:
@veronicagg yes they are.

jimmyca15 commented:
Hey @veronicagg, I'm not really familiar with this CI system. Are these errors that I need to address before you can complete this?
